Modules/Relationships/src/Relationships.Application/Relationships/DTOs/RelationshipChangeDTO.cs

```diff
@@ -7,12 +7,12 @@ namespace Backbone.Modules.Relationships.Application.Relationships.DTOs;
 
 public class RelationshipChangeDTO : IMapTo<RelationshipChange>
 {
-    public RelationshipChangeId Id { get; set; }
+    public required RelationshipChangeId Id { get; set; }
 
-    public RelationshipId RelationshipId { get; set; }
+    public required RelationshipId RelationshipId { get; set; }
 
-    public RelationshipChangeRequestDTO Request { get; set; }
-    public RelationshipChangeResponseDTO Response { get; set; }
+    public required RelationshipChangeRequestDTO Request { get; set; }
+    public required RelationshipChangeResponseDTO Response { get; set; }
 
     public RelationshipChangeType Type { get; set; }
```

Comment:
I think `Type` and `Status` should also be made `required`, since the `RelationshipChange` constructor sets values for those properties.